app/controllers/concerns/nsm/assignment_concern.rb

```diff
@@ -0,0 +1,13 @@
+module Nsm
+  module AssignmentConcern
+    extend ActiveSupport::Concern
+
+    def assign(claim, comment: nil)
```

Comment:
Doesn't `comment:` default to nil?

Reply:
Ah, no, `comment:` makes `comment` a required argument - it will blow up if you try to call `assign` without explicitly providing a `comment`